libs/Database.js

```diff
@@ -632,6 +631,9 @@ class Database {
               // not exist.
               console.log(`Index ${JSON.stringify(ind)} not available for ${finalTableName}`); // eslint-disable-line no-console
             }
+            if (sort.length === 0 || sort[sort.length - 1][0] !== '_id') {
```

Comment:
I don't understand why this logic is included:  `sort[sort.length - 1][0] !== '_id'`

Can you clarify the intent?  If you are aiming to add _id to the sort list if it isn't already in there, why just check the last element instead of all of them?  What if _id is in position 0 instead of position `sort.length - 1` ?

Reply:
Will change to check all, for some reason I thought I needed it as a tiebreak even if specified earlier in the sort order but it's not needed

Reply:
Thanks, looks good with the change.  Good idea to include it as a tiebreak, but there's no need for it to be potentially specified twice in the sort order.